src/casp/services/db/models/ml_management.py

```diff
@@ -28,7 +28,7 @@ class CASMatchingEngineIndex(db.Base):
     index_name = sa.Column(sa.String(255), unique=True, nullable=False)
     embedding_dimension = sa.Column(sa.Integer, nullable=False)
     endpoint_id = sa.Column(sa.String(255), unique=False, nullable=False)
-    deployed_index_id = sa.Column(sa.String(255), unique=True, nullable=False)
+    deployed_index_id = sa.Column(sa.String(255), unique=True, nullable=True)
```

Comment:
I think you're right. I'd honestly love to have active/not-active boolean in the Model model (Model model, lol) so that we can set up a default filter to exclude all non-active models to frontend API to keep them only internally for our needs.

Reply:
Ooh, that's a great call!  Wanna do that as part of this PR?